### DISCUSSION TITLE
Changing `image_scale` value in plotter changes appearance of point labels in screenshots!

### DISCUSSION THREAD

**Discussion by theganeshpandey** [pyvista/pyvista#5215] [Q&A]
Hi All,

I am experiencing an issue while taking screenshots using plotter.screenshot(). While using a pyvista.plotter(image_scale=4) the point labels start to disappear.  Could we mitigate this problem by any means?

A simple example is below which is taken from pyvista documentation with slight modification on `image_scale` paramter:
```python
import numpy as np
import pyvista

pl = pyvista.Plotter(image_scale=4)
points = np.array(
    [[0.0, 0.0, 0.0], [1.0, 1.0, 0.0], [2.0, 0.0, 0.0]]
)
labels = ['Point A', 'Point B', 'Point C']
actor = pl.add_point_labels(
    points,
    labels,
    italic=True,
    font_size=20,
    point_color='red',
    point_size=20,
    render_points_as_spheres=True,
    always_visible=True,
    shadow=True,
)
pl.camera_position = 'xy'
pl.screenshot('3points_scale4.png', return_img=False)
```
The above code does save the image with 3 points but without the points' label. Why?
If we put the default value, i.e. `image_scale=1`, it will save points with all labels.

**Comment by chapochn** [comment]
I also have the same issue. The only workaround I found was to increase the window_size instead of the scale:
```python
ws = np.array([1024, 768], dtype=int)
pl = pyvista.Plotter(window_size=ws*4)
```

Looks like a bug to me.

**Comment by chapochn** [comment]
I actually reported this issue some time ago:
https://github.com/pyvista/pyvista/issues/5748
but it seems like it wasn't resolved yet...
